flow/frame_timings.h

```diff
@@ -51,11 +51,24 @@ class FrameTimingsRecorder {
   /// Timestamp of the vsync signal.
   fml::TimePoint GetVsyncStartTime() const;
 
-  /// Timestamp of when the frame was targeted to be presented.
+  /// Timestamp of the deadline for presenting the frame to the platform
+  /// compositor to avoid missing frames.
   ///
-  /// This is typically the next vsync signal timestamp.
+  /// This is often the next vsync signal timestamp, but may also be
+  /// before or after that time, depending on platform compositor scheduling.
```

Comment:
Should this be named VsyncDeadline?

Reply:
I decided not to rename it as this turns into the timeline attribute "TargetTime" referred to in `flutter_driver`.

Reply:
Why not fix the name here in C/C++ but preserve the timeline attribute? 

Or better, fix both and update flutter_driver (but could be done in a separate CL).